release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2025,6 +2025,18 @@ $ rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
 
 * Currently, on clusters with IPsec enabled, {op-system-base-full} 7.9 nodes cannot communicate with {op-system-first} nodes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925925[BZ#1925925])
 
+* Updates to openvswitch ("net: openvswitch: reorder masks array based on usage") code, causing the openvswitch et/openvswitch/flow_table::flow_lookup accessing per-cpu data on preemptible (and migratable) sections, leading to a real time kernel panic. As a result, the kernel-rt is unstable and will impact low latency applications.
```

Comment:
I don't understand what this first sentence is trying to say. You might want to break it up for clarity.

Reply:
Yes, it is a little problematic, but I was told by Eric that this is understandable to the audience.

Reply:
@erich@redhat.com Can we change this to  "...causes the openvswitch et/openvswitch/flow_table::flow_lookup accessing per-cpu data condition on preemptible (and migratable) sections..."? Or, Am I reading this wrong?

Reply:
Making the above change as per Eric Rich.